api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd

```diff
@@ -1,5 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you remove this intentionally?

Reply:
I did, from my understanding it was unnecessary. Do you want me to add it back to the dtd files?

Reply:
It has been brought to my attention that the DTD files do, in fact, need the XML version and encoding line. Therefore, I have added the line back and pushed my changes again.

Reply:
@dkayiwa besides that does the code look good?